go/sentry/api/grpc.go

```diff
@@ -22,6 +24,12 @@ var (
 	// methodGetUpstreamTLSPubKeys is the GetUpstreamTLSPubKeys method.
 	methodGetUpstreamTLSPubKeys = serviceName.NewMethod("GetUpstreamTLSPubKeys", nil)
 
+	// methodUpdatePolicies is the UpdatePolicies method.
+	methodUpdatePolicies = serviceName.NewMethod("UpdatePolicies", grpcPolApi.ServicePolicies{})
+
+	// methodGetPolicyCheckers is the GetPolicyCheckers method.
+	methodGetPolicyCheckers = serviceName.NewMethod("GetPolicyCheckers", nil)
```

Comment:
How can this be exposed via gRPC? It returns a `*policy.DynamicRuntimePolicyChecker` which is not serializable? This method should probably be removed from the gRPC API.

I assume this method is needed for local use on the sentry node itself? To make this more clear, the sentry `Backend` interface should also be split into two interfaces:
* `Backend` containing all the remotely accessible methods.
* `LocalBackend` that includes `Backend` but also additional methods like `GetPolicyCheckers`.

The gRPC `RegisterService` and `NewSentryClient` would then take/return a `Backend` while the sentry node would keep a `LocalBackend`.